### PR TITLE
Update brew docs

### PR DESCRIPTION
**Pull request by igas**
I renamed formula in brew, after https://github.com/Homebrew/homebrew-core/pull/34283 merged we can update docs.

### CODE REVIEW

**Comment by bw-yuhogg** [comment]
The Homebrew PR has been merged :)

File: docs/content/documentation/getting-started/installation.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ download previous versions of Gutenberg on your package manager of choice in the
 Gutenberg is available on [Brew](https://brew.sh):
 
 ```bash
-$ brew install gutenberg
+$ brew install zola
```
Comment (Keats) [inline docs/content/documentation/getting-started/installation.md]: This section is for gutenberg, can you cut and paste the `macOS` subsection and move it above in the Zola section?

File: docs/content/documentation/getting-started/installation.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ weight = 1
 Zola provides pre-built binaries for MacOS, Linux and Windows on the
 [GitHub release page](https://github.com/getzola/zola/releases).
 
+### macOS
+
+Gutenberg is available on [Brew](https://brew.sh):
```
Comment (Keats) [inline docs/content/documentation/getting-started/installation.md]: Zola :p

**Comment by Keats** [comment]
Thanks!

**Pull request merged**